app/Console/Commands/CloudCheckSubscription.php

```
@@ -27,23 +30,290 @@ class CloudCheckSubscription extends Command
+            if ($checkOne) {
+                $this->info('Checking and fixing only the first canceled subscription...');
+            } else {
+                $this->info('Checking and fixing canceled subscriptions...');
+            }
+        }
+
+        $teamsWithSubscriptions = Team::whereRelation('subscription', 'stripe_invoice_paid', true)->get();
+        $toFixCount = 0;
+        $fixedCount = 0;
+        $errors = [];
+        $canceledSubscriptions = [];
```

PHPMD (2.15.0)

Avoid excessively long variable names like $canceledSubscriptions. Keep variable name length under 20. (undefined)

(LongVariable)